study-builder/studies/testboston/snippets/question-adhoc-symptoms.conf

```diff
@@ -0,0 +1,226 @@
+              {"language": "es", "text": ${i18n.es.option.fatigue}},
+              {"language": "ht", "text": ${i18n.ht.option.fatigue}}
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "stableId": "MUSCLE_ACHE",
+      "optionLabelTemplate": {
+        "templateType": "TEXT",
+        "templateText": "$adhoc_symptoms_muscle_ache",
+        "variables": [
+          {
+            "name": "adhoc_symptoms_muscle_ache",
+            "translations": [
+              {"language": "en", "text": ${i18n.en.option.muscle_ache}},
+              {"language": "es", "text": ${i18n.es.option.muscle_ache}},
+              {"language": "ht", "text": ${i18n.ht.option.muscle_ache}}
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "stableId": "SENSORY_LOSS",
+      "optionLabelTemplate": {
+        "templateType": "TEXT",
+        "templateText": "$adhoc_symptoms_sensory_loss",
+        "variables": [
+          {
+            "name": "adhoc_symptoms_sensory_loss",
+            "translations": [
+              {"language": "en", "text": ${i18n.en.option.sensory_loss}},
+              {"language": "es", "text": ${i18n.es.option.sensory_loss}},
+              {"language": "ht", "text": ${i18n.ht.option.sensory_loss}}
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "stableId": "NAUSEA",
+      "optionLabelTemplate": {
+        "templateType": "TEXT",
+        "templateText": "$adhoc_symptoms_nausea",
+        "variables": [
+          {
+            "name": "adhoc_symptoms_nausea",
+            "translations": [
+              {"language": "en", "text": ${i18n.en.option.nausea}},
+              {"language": "es", "text": ${i18n.es.option.nausea}},
+              {"language": "ht", "text": ${i18n.ht.option.nausea}}
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "stableId": "CONGESTION",
+      "optionLabelTemplate": {
+        "templateType": "TEXT",
+        "templateText": "$adhoc_symptoms_congestion",
+        "variables": [
+          {
+            "name": "adhoc_symptoms_congestion",
+            "translations": [
+              {"language": "en", "text": ${i18n.en.option.congestion}},
+              {"language": "es", "text": ${i18n.es.option.congestion}},
+              {"language": "ht", "text": ${i18n.ht.option.congestion}}
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "stableId": "HEADACHE",
+      "optionLabelTemplate": {
+        "templateType": "TEXT",
+        "templateText": "$adhoc_symptoms_headache",
+        "variables": [
+          {
+            "name": "adhoc_symptoms_headache",
+            "translations": [
+              {"language": "en", "text": ${i18n.en.option.headache}},
+              {"language": "es", "text": ${i18n.es.option.headache}},
+              {"language": "ht", "text": ${i18n.ht.option.headache}}
+            ]
+          }
+        ]
+      }
+    }
+  ],
+  "validations": [
+    {
+      "ruleType": "REQUIRED",
```

Comment:
User will submit survey only when they see some symptoms.
workaround to initial requirement:
"This activity is write-once; read-always.
However, if this activity is "submitted" with 0 answers provided, then it should not be marked as "complete" and it should remain editable/writeable."